src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -424,10 +424,8 @@ def geoprocessing_chains(aoi, wkaoi, exchange, errback, choose_worker):
     ]
 
     return [
-        start.s(opname, data, wkaoi).set(exchange=exchange,
-                                         routing_key=worker) |
-        finish.s().set(exchange=exchange,
-                       routing_key=worker) |
+        run.s(opname, data, wkaoi).set(exchange=exchange,
+                                       routing_key=worker) |
```

Comment:
@hectcastro We do have https://github.com/WikiWatershed/model-my-watershed/issues/2117 to address that, but it'd be helpful if you could comment on the statement I made in there about dark stack routing - that was my recollection, but I wasn't confident it was accurate.